gk/main.c

```diff
@@ -1789,7 +1789,7 @@ process_cmds_from_mailbox(
         }
 }
 
-static void
+void
 send_pkts(uint8_t port, uint16_t tx_queue,
 	uint16_t num_pkts, struct rte_mbuf **bufs)
```

Comment:
This function is generic and doesn't rely on the GK block. And now that the LLS block is using it as well, move it to a more proper file.